onnx/cpp2py_export.cc

```diff
@@ -96,6 +97,57 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
       .value("COMMON", OpSchema::SupportType::COMMON)
       .value("EXPERIMENTAL", OpSchema::SupportType::EXPERIMENTAL);
 
+  py::class_<FunctionProto> function_proto(defs, "FunctionProto");
```

Comment:
This is not good, because python land already has a `FunctionProto` class from the auto-generated protobuf python files (`onnx.FunctionProto`), and its underlying memory layout is totally different from the c++ protobuf structs, and so python land users will see two classes with the same name while they are completely different. what we have been doing when passing protobuf structs across c++ and python is serialize on one side and then deserialize on the other side (look at check_model)

Reply:
Currently the solution is blocked by #1194 (and seems the import issue is in discussion in protobuf repo as well). I could rename the generated pybind class (to function_schema probably) and explore a way to resolve #1194 (probably need to manually run text replacement scripts), switch to use function_proto later. This class is only used for the display of function information now so there is use case now for user to really create one using function_proto class in _pb2.py

Reply:
Found a way to manually correct the import in build. Pushed.

Reply:
Comment resolved